bot/src/infrastructure/http/zkill.ts

```
@@ -1,130 +1,292 @@
+  initialTimeout: 15000, // Initial timeout of 15 seconds
+  maxTimeout: 45000, // Maximum timeout of 45 seconds
+};
 
 /**
  * Client for interacting with ZKillboard API
  */
 export class ZKillboardClient {
   private readonly client: AxiosInstance;
-  private readonly config: Required<ZKillboardClientConfig>;
+  private readonly baseUrl: string;
+  private lastRequestTime: number = 0;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)